src/test/java/io/github/sskorol/testcases/XlsxDataSupplierTests.java

```diff
@@ -29,4 +34,9 @@ public void shouldReadLocalExcelSpreadsheetWithoutSheet(final PersonWithoutSheet
     public void shouldReadLocalExcelSpreadsheetWithSheet(final PersonWithSheet person) {
         // not implemented
     }
+
+    @Test(dataProvider = "getDataWithCustomConverters")
+    public void shouldReadLocalExcelSpreadsheetWithCustomConverters(final TestData testData){
+        System.out.println(testData);
```

Comment:
`System.out.println` is redundant here and violated in static code analysis tools.